test/misc/bigfloat_test.jl

```diff
@@ -39,6 +39,6 @@
     u0 = BigFloat.([1.0, 1.0, 1.0])
     tspan = (0.0, 1.0)
     prob = SecondOrderBVProblem(second_f!, second_bc!, u0, tspan)
-    sol4 = solve(prob, MIRKN4(), dt = 0.01)
-    @test SciMLBase.successful_retcode(sol4.retcode)
+    @test_broken sol4 = solve(prob, MIRKN4(), dt = 0.01)
+    @test_broken SciMLBase.successful_retcode(sol4.retcode)
```

Comment:
What happened with bigfloat support?

Reply:
It's failing because of the missing dispatches for SparseArrays factorization in LinearSolve.jl, I am digging into what's going wrong since it was doing fine before🤔